docs/gameplay-sessions/gameplay-sessions-usage.mdx

```diff
@@ -34,6 +34,55 @@ Once you've selected a replay, click the Play button below the Regression Games
 ![Open the Gameplay Session Selector](img/overlay_click_load.png)
 ![Select the Gameplay Session](img/overlay_choose_session.png)
 
+## Programmatically Playing a Recording
+
+A recorded gameplay session can also be played back programmatically, using a 
+[Unity Test Runner Play Mode test](https://docs.unity3d.com/Packages/com.unity.test-framework@1.4/manual/index.html).
+After creating a Play Mode test file, the `RGTestUtils` class can be used to load and play a recording, which in turn
+will also record a new session for that test.
+
+:::info Note
+
+Recording `data.zip` files are saved by default to `~/unity_videos/{Application.productName}/run_{#}/`. Make sure
```

Comment:
This path isn't correct

Reply:
Fixed here now and in the other locations where I copied this from. Since the new definition is quite long, I chose to define it in one spot and then reference it in other locations